topics/ingest-feedback-events.yaml

```diff
@@ -0,0 +1,15 @@
+topic: ingest-feedback-events
+pipeline: user-feedback
+description: User feedback events from SDK
+services:
+  producers:
+    - getsentry/relay
+  consumers:
+    - getsentry/sentry
+schemas:
+  - version: 1
+    compatibility_mode: none
+    type: json
+    resource: ingest-feedback-events.v1.schema.json
+    examples:
+      - ingest-feedback-events/1/
```

Comment:
One more thing -- let's add the `max.message.bytes` for this topic too? We don't want a repeat of today's INC-670 where no one is sure what the topic's max size is.